mantidproject/mantid
# Improve Muon fitting refactor release notes

**Description of work.**
This PR improves the muon release notes regarding the recent refactor of the fitting tab. It adds an image showing the new design of the fitting tab.

**To test:**
Build the docs-html target and check that the new image in the muon release notes looks ok.

Alternatively, you might be able to just click `View File` in the Files changed section of this PR and it shows you the new image.

*There is no associated issue.*

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
